### ISSUE TITLE
Handling lang code "zxx"

### ISSUE THREAD

**Issue by sdigiulio** [Brown-University-Library/usep-data#11]
The language code "zxx", used for non-linguistic data, currently validates within EpiDoc and our inscriptions; however, our indexing doesn't process it and it triggers a 500 server error. We should make sure 1) indexing handles it properly and 2) that the xsl also handles it properly.

**Comment by emylonas** [comment]
can't find this value in any file.

**Comment by jbodel** [comment]
If lang. code "zxx" is not in use, shall we close this issue? We DO want to make sure our new lang., codes will index properly. Will this work with non Unicode lang. abbreviations, such as "Raet" (= Raetic)

**Comment by sdigiulio** [comment]
All language codes in the metadata fields should use the ISO-639 codes, which include Raetic as "xrr". In terms of using it in the USEP ID, using "Raet" is fine -- it's in the metadata for `@mainLang` that using something other than the ISO codes would cause problems.

Based on what was previously indexed, it seems that we did have inscriptions using both codes, though I agree with Elli that the former doesn't seem to be showing up in what is currently in Git. It's possible that I pulled them out after they were causing issues, and I can check where the files using these codes are.
